packages/web-components/src/components/ic-button/ic-button.tsx

```diff
@@ -199,6 +209,11 @@ export class Button {
    */
   @Event() icBlur!: EventEmitter<void>;
 
+  /**
+   * When 'fileUpload' is set to 'true', this will be emitted when a file is selected in the native explorer.
```

Comment:
I think use "If" instead of "When" here to match the prop description on line 68 as we want them all to be worded as consistently as possible :)